spec/features/locales_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe 'locale' do
+
+  context 'when arabic is selected' do 
+    
+    it 'works' do
+      visit root_path(:en)
+      find("#change-to-arabic").click
```

Comment:
Tests that assert that a page loads are too weak when you introduce interactions such as `find("#change-to-arabic").click`.

Instead (or additionally), assert that clicking the button does something.

Probably: `expect{ find("#change-to-arabic").click }.to change{ current_url }.from(x).to(y)`